lib/gssapi/mech/gss_names.c

```diff
@@ -38,6 +40,10 @@ _gss_find_mn(OM_uint32 *minor_status, struct _gss_name *name, gss_OID mech,
 
 	*output_mn = NULL;
 
+	/* null names are ok, some mechs might not have names */
```

Comment:
I can see that we might support `GSS_C_NO_NAME` as a `targ_name` in `gss_init_sec_context()`, but is it wise to make this choice so globally?  Might we paper over bugs this way?